gstd/src/msg/macros.rs

```diff
@@ -64,6 +64,44 @@ macro_rules! impl_futures {
 
                 Ok(self)
             }
+
+            /// Execute a function when the reply is received.
+            ///
+            /// Note: This callback will run in reply context and consume reply gas.
```

Comment:
Probably you also want to mention it is called before the continuation if the latter is still awaiting execution